scripts/generate_userguide_epub.py

```
@@ -0,0 +1,321 @@
+    pattern = re.compile(r'(<h2[^>]*>)(.*?)(</h2>)', re.DOTALL)
+    chapters = []
+    positions = [(m.start(), m.group(0), m.group(2)) for m in pattern.finditer(html)]
+
+    if not positions:
+        return [('intro', 'CrossPoint User Guide', html)]
+
+    intro_html = html[:positions[0][0]].strip()
+    if intro_html:
+        chapters.append(('intro', 'Introduction', intro_html))
+
+    for idx, (start, tag, raw_title) in enumerate(positions):
```

Ruff (0.15.14)

[warning] Loop control variable `tag` not used within loop body

Rename unused `tag` to `_tag`

(B007)